src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,5 @@
-const numberToReversedDigits = number => {};
+const numberToReversedDigits = number => splitArray = number.toString().split("").map(Number).reverse();
+    
```

Comment:
You don't need this splitArray, you can return the `number.toString()...` directly.